composer.json

```diff
@@ -18,15 +18,14 @@
         "composer/semver": "^3.4",
         "composer/xdebug-handler": "^3.0.3",
         "doctrine/inflector": "^2.0.9",
-        "fidry/cpu-core-counter": "^1.1",
         "illuminate/container": "^10.43",
+        "nette/robot-loader": "^4.0",
```

Comment:
RobotLoader is only used on `rector-src` which run test along with `rector-*` extension, and currently not used in scoped rector/rector, so should be kept on `require-dev`, for now

Reply:
It is currently present in `require-dev`, but used in production path `utils` (present in `autoload`, not `autoload-dev`). That is why it was detected as miscategorized. I can ignore it, but maybe `utils` belongs to dev paths actually?

Reply:
`utils` dir needs in `require`, as it used during downgrade process, which if use require-dev, it already not autoloaded.